### DISCUSSION TITLE
WPP não avança ao inicializar

### DISCUSSION THREAD

**Discussion by joaquimCFC** [wppconnect-team/wppconnect#2039] [Q&A]
Ola, primeira vez aqui,

apos algumas atulizações o wpp conect nao esta inicializando

![image](https://github.com/wppconnect-team/wppconnect/assets/132825083/c0b9ddbd-2fa6-422b-a0c5-08ef7e501a73)

paralisando neste etagio

info:     [whatsbot:browser] Initializing browser...

**Comment by LuizVeloso10** [comment]
wppconnect. create 

coloque a opção 

puppeteerOptions: {
					 headless: 'new',
				},

dessa forma deve funcionar !
